apis/spec/v1alpha2/servicebinding_types.go

```diff
@@ -33,7 +34,7 @@ type ServiceBindingApplicationReference struct {
 	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
 	Name string `json:"name,omitempty"`
 	// Selector is a query that selects the application or applications to bind the service to
-	Selector metav1.LabelSelector `json:"selector,omitempty"`
+	Selector *metav1.LabelSelector `json:"selector,omitempty"`
```

Comment:
what is the motivation for this change?

Reply:
This is required to support empty values similar to the non-spec API.

Reply:
not following - spec API already has support for referring an api via name or through application selector. What ServiceBinding resource do you have in mind? Can you please share the example?

Reply:
I am trying to follow this :
https://github.com/redhat-developer/service-binding-operator/blob/master/apis/binding/v1alpha1/servicebinding_types.go#L153
We are using a pointer there.

Reply:
With that, we are drifting from spec API, right? We should submit this change to the spec first. Can we implement the verification with that change for now?

Reply:
spec updated.